mixins/localize/README.md

```diff
@@ -144,7 +93,13 @@ Remember that `<strong>` is for content of greater importance (browsers show thi
 
 Similarly `<em>` *emphasizes* a particular piece of text (browsers show this visually using italics), whereas `<i>` only italicizes the text visually without emphasis.
 
-### Links
+```json
+{
+  "myTerm": "This is <b>bold</b> but <em>not</em> all that <strong>important</strong>."
+};
+```
```

Comment:
@svanherk specific example based on your feedback in demos yesterday

Reply:
Thanks! Does it make sense to also show that the usage is just `this.localizeHTML('myTerm')` with nothing passed in as the second parameter?  Maybe that's overkill

Reply:
I think that could be helpful 👍.